locales/de/de.json

```diff
@@ -430,7 +430,7 @@
     "Restore Resource": "Ressource wiederherstellen",
     "Restore Selected": "Wiederherstellen Ausgewählte",
     "results": "Ergebnisse",
-    "Reunion": "Réunion",
+    "Reunion": "Wiedervereinen",
```

Comment:
Sorry, but that's completely wrong.  Please also note the context used.  This complete pull request makes it worse than before.

https://de.m.wikipedia.org/wiki/R%C3%A9union

Reply:
Here, it's the country, it's not "Réunion" ?

Reply:
Missunderstanding from my side here, totally agree on you